arches/app/utils/betterJSONSerializer.py

```diff
@@ -53,7 +53,11 @@ def __init__(self, **options):
         self.utf_encode = False
 
     def encode(self, obj):
-        return self.serializeToPython(obj, **self._options)
+        if isinstance(obj, (I18n_JSON, I18n_String)):
+            # Ensure raw JSON strings are re-encoded ("'" -> "\"")
+            # and non-active language values are preserved.
+            return DjangoJSONEncoder().encode(obj.raw_value)
+        return self.serialize(obj, **self._options)
```

Comment:
`serializeToPython()` doesn't seem to return strings in every case, so I backed out to `serialize()`, which does. That clarified why I can't just `serialize()` for the I18n fields -- `serialize()` refuses to re-encode, but we _need_ to re-encode in this case to preserve double quotes.